scripts/common/flags.sh

```diff
@@ -214,6 +214,7 @@ function flags() {
                 # allow either /16 or 16 for subnet size
                 SERVICE_CIDR_RANGE=$(echo "$_value" | sed "s/\///")
                 ;;
+            # keep this
```

Comment:
What does this comment mean?

Reply:
this is one of the things going to be kept in flags.sh, will be removed shortly